components/director/internal/domain/application/service.go

```diff
@@ -301,6 +303,21 @@ func (s *service) Delete(ctx context.Context, id string) error {
 		return errors.Wrapf(err, "while loading tenant from context")
 	}
 
+	scenarios, runtimes, err := s.isInScenarioAndRuntime(ctx, appTenant, id)
```

Comment:
Is it necessary to check for the runtimes?  I personally think that we can skip this because the runtimes in a given formation can be easily seen from the UI.

Reply:
Same as https://github.com/kyma-incubator/compass/pull/1769#discussion_r599410457